mediorum/server/db.go

```diff
@@ -40,6 +40,26 @@ type Upload struct {
 	// UpldateULID - this is the last ULID that change this thing
 }
 
+// Metric actions
+const (
+	StreamTrack string = "stream_track"
+	ServeImage  string = "serve_image"
+)
+
+type DailyMetrics struct {
+	Timestamp time.Time `gorm:"primaryKey"`
+	Action    string    `gorm:"primaryKey"`
+	Count     int64     `gorm:"not null"`
+	CreatedAt time.Time `json:"created_at" gorm:"not null"`
```

Comment:
what's the purpose of created_at here and in MonthlyMetrics?

Reply:
no purpose really, just for potential debugging